model/textparse/openmetricsparse.go

```diff
@@ -511,6 +627,21 @@ func (p *OpenMetricsParser) parseMetricSuffix(t token) (Entry, error) {
 			return EntryInvalid, p.parseError("expected next entry after timestamp", t3)
 		}
 	}
+
+	if p.skipCT {
```

Comment:
This could be improved (: 

We do extra, important recursive call to `p.Next` from a low level function that suppose to only parse suffix. Wouldn't caller be surprise that on top of parsing suffix we also parse metrics and for some types we skip the whole entry? 🤔 smells like spaghetti and someone's frustration later on ;p

Probably better approach would be to have
* a clear function that determines what is CT series (this is important because per OM spec this can go wrong -- conflicts) e.g. `IsCTSeries(...)`
* I would use this strictly in `Next` to control this recursion with care and transparency to the next code reader.

WDYT?